src/app/demo/bootstrap/bootstrap.component.ts

```diff
@@ -11,12 +11,12 @@ export class BootstrapComponent {
 
   code = {
     decorators: {
-      code: `// @Component is a decorator
+      code: `// @Component is an Angular decorator
 @Component({
   metadata
-})
+}) // No semicolon here!
```

Comment:
should we add a line explaining why? e.g. "so it can attach to the component" or something